reactDashboard/src/components/DashboardNavbar/DashboardNavbar.jsx

```diff
@@ -21,20 +24,38 @@ const DashboardNavbar = () => {
   }
 
   return (
-    <Navbar className="dashboard-navbar" bg="white" expand="sm">
+    <Navbar className="dashboard-navbar" expand="sm">
       <Link to={URL_OVERVIEW}>
-        <img id="logo" src="/STALogo.png" alt="Scottish Tech Army Logo" />
+        <img
+          id="logo"
+          src="/STALogoSquare.svg"
+          alt="Scottish Tech Army Logo"
+        />
       </Link>
+      <Nav className="hide-darkmode">
+      <FontAwesomeIcon
```

Comment:
I know this duplication was my suggestion, but... ;)
How about extracting this element out into a function darkmodeIcon() and calling it in the 2 places ?

Reply:
✅